ospf6d/ospf6_top.c

```diff
@@ -962,12 +962,13 @@ DEFUN_HIDDEN (ospf6_interface_area,
 	oi->area_id_format = format;
 
 	oa = ospf6_area_lookup(area_id, ospf6);
-	if (oa == NULL)
+	if (oa == NULL) {
```

Comment:
The additional check is not needed in this function, it already checks that the `oi->area` is NULL before.